tensorflow/tools/tf_sig_build_dockerfiles/devel.requirements.txt

```diff
@@ -4,23 +4,23 @@
 # To have reproducible builds, these dependencies should be pinned always.
 # Prefer pinning to the same version as in setup.py for now.
 # This will change in the future.
-absl-py ~= 1.0.0
-astunparse ~= 1.6.3
-flatbuffers ~= 2.0
-google_pasta ~= 0.2
-h5py ~= 3.8.0 # Earliest version for Python 3.11
+absl-py == 1.0.0
+astunparse == 1.6.3
+flatbuffers == 23.1.21
```

Comment:
This seems wrong. From 2.0 to 23.1.21?

Reply:
so can i use the same 2.0 ?

Reply:
This is the history for flatbuffers. https://github.com/tensorflow/tensorflow/commits/master/tensorflow/tools/ci_build/release/requirements_common.txt

Reply:
You have to use the python wheel version number

Reply:
As for what number to use, you have to see what gets installed in the build log for the last nightly and use that one